index.html

```diff
@@ -51,15 +51,15 @@ <h1 class="intro_greeting">I'm George<br> Nice to see you!</h1>
 
 			<section class="Works">
 				<div class="work_item">
-					<img src="images/Snapshoot Portfolio.png" alt="snapshot portifolio image">
+					<img src="images/Snapshoot Portfolio.png" alt="snapshot portifolio image" class="portifolio_img">
 					<div class="work_details">
 						<div class="project">
 							<h2 class="work_item_header">Tonic</h2>
```

Comment:
- [ ] Great job working on your projects section, It looks great! 👏🏼 However, in order to stick as much as possible to the Figma design 🎨 I would suggest some minor fixes such as the font size of the title, the button styles, and the button interaction effects such as hover, pressed, etc. 🤓 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199998011-9d67986a-8f1c-458e-bb4f-efe4c21b1d1f.png) | ![imageedit_14_2050133455](https://user-images.githubusercontent.com/77069011/199997939-67aec1a0-b22a-41bc-86be-6300731baf4f.gif) |

Button interaction effects:
![Screenshot from 2022-11-01 06-57-41](https://user-images.githubusercontent.com/77069011/199998527-d0e7fa6a-cc2d-433a-bfdd-49cfbdc168ff.png)